src/components/core/PivotTable.tsx

```diff
@@ -1,23 +1,53 @@
 // (C) 2007-2018 GoodData Corporation
 import * as React from 'react';
 import { AgGridReact } from 'ag-grid-react';
+import * as classnames from 'classnames';
 import noop = require('lodash/noop');
+import get = require('lodash/get');
 import isEqual = require('lodash/isEqual');
 import { AFM, Execution } from '@gooddata/typings';
 
 import { visualizationIsBetaWarning } from '../../helpers/utils';
-import { executionToAGGridAdapter, IGridHeader } from '../../helpers/agGrid';
+
+import {
+    executionToAGGridAdapter,
+    IGridHeader,
```

Comment:
how about adding this to interfaces folder?

Reply:
done